### DISCUSSION TITLE
Using Github for package registry only

### DISCUSSION THREAD

**Discussion by lashawnta-pf** [community/community#38012] [Packages]
Our source code is currently housed in Bitbucket for business reasons.    Should I be able to publish a package from Bitbucket CI/CD to Github Packages?   Essentially we would like to just leverage the npm registry capabilities only (at least for the near term).  Are there any obstacles we should expect doing so?

**Comment by david-engelmann** [comment]
You can create a Github repo and design a Bitbucket workflow to commit the code to the Github repo. You'll need to include in the Bitbucket repo the Github authentication files. If you added in the config, public and private rsa files, you'd be able to automatically push changes to BitBucket branches to the identical Github branches

**Comment by lashawnta-pf** [comment]
So a GitHub repository with the source code  is still necessary to use
GitHub packages, even when BitBucket is the primary source code control.
